feat: add config option to reorder items in emoji picker

coder-with-a-bushido opened the pull request.

Fintasys commented on example/lib/main.dart:
```diff
@@ -109,6 +109,11 @@ class MyAppState extends State<MyApp> {
                   config: Config(
                     height: 256,
                     checkPlatformCompatibility: true,
+                    itemsOrderConfig: const ItemsOrderConfig(
+                      top: EmojiPickerItem.categoryBar,
+                      middle: EmojiPickerItem.emojiView,
+                      bottom: EmojiPickerItem.searchBar,
+                    ),
```
Nit: Might would use the term `view` instead of `item`, what do you think?

coder-with-a-bushido replied:
done

Fintasys commented on example/lib/main_whatsapp.dart:
```diff
@@ -191,11 +191,15 @@ class MyAppState extends State<MyApp> {
+                            middle: EmojiPickerItem.emojiView,
+                            bottom: EmojiPickerItem.searchBar,
+                          ),
                           emojiTextStyle: _textStyle,
                           emojiViewConfig: const EmojiViewConfig(
                             backgroundColor: Colors.white,
                           ),
-                          swapCategoryAndBottomBar: true,
```
Updating the ReadMe would be appreciated

coder-with-a-bushido replied:
done

Fintasys commented:
Thank you for your PR - the code looks mostly good. I couldn't imagine that anyone would have the emoji view else where than in the middle, so I would be interested what has motivated you to make this change?

coder-with-a-bushido commented:
> Thank you for your PR - the code looks mostly good. I couldn't imagine that anyone would have the emoji view else where than in the middle, so I would be interested what has motivated you to make this change?

Hey @Fintasys, thanks for the review!
I have addressed the comments now (sorry I was sick for a week, so I couldn't do it any soon).

As for the motivation behind this PR, it's because the design for an app I'm working on demanded this order - category view, search bar, emoji view.

Fintasys reviewed (approved):
LGTM
